docs/source/timing-analysis.rst

```diff
@@ -258,7 +296,7 @@ To run the tests:
 1. Select a machine with sufficient cooling and a multi-core CPU
 2. Use methods mentioned before to create isolated cores, watch out for
    hyperthreading
-3. For RSA tests use small key (1024 bit), for CBC tests use a fast cipher and
+3. For RSA tests use small key (1024 bit), use a fast cipher and
```

Comment:
Either remove `use a` or start a new sentence. This way it sounds weird.

Reply:
done